inc/knowbaseitem.class.php

```diff
@@ -1859,15 +1853,6 @@ function rawSearchOptions() {
          'datatype'           => 'dropdown'
       ];
 
-      $tab[] = [
```

Comment:
You have to take care of use preferences table, and saved searches

Reply:
Done in last commit.